### PR TITLE
fix #301116: don't write default note event values along with non-default ones

### PR DESCRIPTION
**Pull request by Howard-C**
Resolves: https://musescore.org/node/301116.

### CODE REVIEW

File: libmscore/noteevent.cpp [hunk 1]
```diff
@@ -41,9 +41,12 @@ void NoteEvent::read(XmlReader& e)
 void NoteEvent::write(XmlWriter& xml) const
       {
       xml.stag("Event");
-      xml.tag("pitch", _pitch);
-      xml.tag("ontime", _ontime);
-      xml.tag("len", _len);
+      if (_pitch != 0)
+            xml.tag("pitch", _pitch);
```
Comment (Jojo-Schmitz) [inline libmscore/noteevent.cpp]: those `if`s are not needed it using `xml.tag()`'s 3rd agrument

      xml.tag("pitch", _pitch, 0);
      xml.tag("ontime", _ontime, 0);
      xml.tag("len", _len, NOTE_LENGTH);

Comment (Howard-C) [reply libmscore/noteevent.cpp]: Oh

File: libmscore/note.cpp [hunk 2]
```diff
@@ -1395,7 +1395,7 @@ bool Note::readProperties(XmlReader& e)
             while (e.readNextStartElement()) {
                   const QStringRef& t(e.name());
                   if (t == "Event") {
-                        NoteEvent ne;
+                        NoteEvent ne = NoteEvent();
```
Comment (dmitrio95) [inline libmscore/note.cpp]: What is the purpose of this change? `NoteEvent ne;` already invokes `NoteEvent`'s default constructor.

Comment (Howard-C) [reply libmscore/note.cpp]: Oh right
